sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1049, "Unknown database 'mydb'") (Background on this error at: http://sqlalche.me/e/e3q8)

Fix: correctly spell/ have 'host'

NameError: global name 'd' is not defined

Fix: under database make sure you have a defined variable

sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1045, "Access denied for user 'rot'@'localhost' (using password: YES)") (Background on this error at: http://sqlalche.me/e/e3q8)

Fix: Have proper password/username

sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (10061)") (Background on this error at: http://sqlalche.me/e/e3q8)

Fix: Have correct port